code/controllers/master_controller.dm

```diff
@@ -71,6 +71,12 @@ datum/controller/game_controller/proc/setup_objects()
 			T.broadcast_status()
 
 
+	//Spawn the contents of the cargo warehouse
+	admin_notice("<span class='danger'>Populating cargo bay..</span>", R_DEBUG)

```

Comment:
I think these kinds of notices generally work best when they're of the form "populating cargo bay took x seconds". Also it's better when the notice comes from a safe completion check.